contrib/setup

```diff
@@ -22,6 +22,18 @@ rename_branch()
     fi
 }
 
+detect_selinux()
+{
+    SELINUX="0"
+    which getenforce
+    if [ "$?" -eq 0 ]; then
+        getenforce | grep -i "enforcing"
+        if [ "$?" -eq 0 ]; then
```

Comment:
could this be simpler?

```
if getenforce | grep -i "enforcing" 2> /dev/null 1>&2; then
   SELINUX=1
fi
```

I think that if it doesn't exist that fails, and if it's not enforcing that fails, and if it is it passes